dao/OrderDao.py

```
@@ -0,0 +1,269 @@
+#!/usr/bin python3
+# -*- coding: utf-8 -*-
+"""
+@CreateTime    : 2024/5/6 14:30
+@Author  : 系统订单服务
+@file for: 订单数据访问层
+"""
+import traceback
+from psycopg2 import sql
+from psycopg2.extras import execute_values
+
+from logger_control import SharingProgramInfo, log_info
```

Ruff (0.11.9)

`logger_control.SharingProgramInfo` imported but unused

Remove unused import: `logger_control.SharingProgramInfo`

(F401)